components/base/wheeled/kinematics_test.go

```diff
@@ -75,3 +75,34 @@ func TestWrapWithKinematics(t *testing.T) {
 		})
 	}
 }
+
+func TestCurrentInputs(t *testing.T) {
+	ctx := context.Background()
+	logger := golog.NewTestLogger(t)
+	base := &wheeledBase{
+		widthMm:              400,
+		wheelCircumferenceMm: 25,
+		logger:               logger,
+		name:                 "count basie",
+		frame: &referenceframe.LinkConfig{
+			Parent: referenceframe.World,
+			Geometry: &spatialmath.GeometryConfig{
+				R: 400,
+			},
+		},
+	}
+
+	kb, err := base.WrapWithKinematics(ctx, fake.NewSLAM(resource.NewName(slam.API, "foo"), logger))
+	test.That(t, err, test.ShouldBeNil)
+	kwb, ok := kb.(*kinematicWheeledBase)
+	test.That(t, ok, test.ShouldBeTrue)
+	for i := 0; i < 100; i++ {
```

Comment:
What is the purpose of this loop?

Reply:
Fake slam will update both map and position when GetPointCloudMapFull is called, so it isn't ideal for testing localization mode, where the map shouldn't change. However, I think it's fine if we just ignore changes to the map for now.

I was just confused why we are iterating 100 times through the map/position here.

Reply:
This was a test that I forgot to clean up.  I did want to loop through several times to make sure we are getting info from slam correctly though

Reply:
ah got it :)

Reply:
Changed this to something saner